src/tir/transforms/lower_tvm_builtin.cc

```diff
@@ -113,6 +113,16 @@ class BuiltinLower : public StmtExprMutator {
     op = stmt.as<AllocateNode>();
     // Get constant allocation bound.
     int64_t nbytes = GetVectorBytes(op->dtype);
+    if (device_type_.defined()) {
```

Comment:
Hi, @mbrookhart 

Can you explain the rationale for not generating TVMBAW calls for static size allocates for DLCPU ?

cc: @grant-arm @Mousius @mbaret